packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceConfigSet.test.ts

```diff
@@ -6,34 +6,35 @@
  */
 
 import { expect } from 'chai';
+import * as sinon from 'sinon';
+import { createSandbox } from 'sinon';
+import * as vscode from 'vscode';
 import { ForceConfigSetExecutor } from '../../../src/commands';
-import { nls } from '../../../src/messages';
+
+const env = createSandbox();
```

Comment:
nit: can you rename env -> sandbox. It would just be more consistent with existing code.